tests/unit/test_cli.py

```diff
@@ -90,17 +90,26 @@ def test_get_text(cli_runner, vault_with_token, extra_args):
     vault_with_token.db = {"a": {"value": "bar"}}
     result = cli_runner.invoke(cli.cli, ["get", "a"] + extra_args)
 
+    assert result.output == "---\nvalue: bar\n"
+    assert result.exit_code == 0
+
+    result = cli_runner.invoke(cli.cli, ["get", "a", "value"] + extra_args)
+
     assert result.output == "bar\n"
     assert result.exit_code == 0
 
 
 @pytest.mark.parametrize(
-    "input, output",
-    [([1, 2], "---\n- 1\n- 2\n"), ({"a": "b"}, "---\na: b\n"), (None, "null\n")],
+    "value, output",
+    [
+        ({"list": [1, 2]}, "---\nlist:\n- 1\n- 2\n"),
+        ({"a": "b"}, "---\na: b\n"),
+        (None, "null\n"),
```

Comment:
```suggestion
        ({"key": None}, "---\nkey: null\n"),
```

Reply:
the test was good, we only change the value for the key "a" and not the mapping, it is used to test non-string values